app/src/main/java/com/github/sdpsharelook/TranslateActivity.kt

```diff
@@ -0,0 +1,109 @@
+package com.github.sdpsharelook
+
+import android.os.Bundle
+import android.view.View
+import android.widget.*
+import androidx.annotation.Nullable
+import androidx.annotation.VisibleForTesting
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.widget.addTextChangedListener
+import androidx.test.espresso.IdlingResource
+import androidx.test.espresso.idling.CountingIdlingResource
+import com.github.sdpsharelook.translate.Translator
+import com.google.mlkit.nl.translate.TranslateLanguage
+import kotlinx.coroutines.*
+
+
+class TranslateActivity : AppCompatActivity() {
+    // They array of all languages available took from TranslateLanguage.allLanguages()
+    private val allLanguages = arrayOf("af", "sq", "ar", "be", "bg", "bn", "ca", "zh", "hr", "cs",
+        "da", "nl", "en", "eo", "et", "fi", "fr", "gl", "ka", "de", "el", "gu", "ht", "he", "hi",
+        "hu", "is", "id", "ga", "it", "ja", "kn", "ko", "lt", "lv", "mk", "mr", "ms", "mt", "no",
+        "fa", "pl", "pt", "ro", "ru", "sk", "sl", "es", "sv", "sw", "tl", "ta", "te", "th","tr",
+        "uk", "ur", "vi", "cy"
+    )
```

Comment:
I think you can use something like that : 
Locale.getAvailableLocales().map { it.toLanguageTag() }
instead of hard-coding each language